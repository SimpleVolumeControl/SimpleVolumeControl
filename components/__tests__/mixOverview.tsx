import { render } from '@testing-library/react';
import MixOverview from '../mixOverview';
import { RecoilRoot } from 'recoil';

jest.mock('next/router', () => ({
  useRouter: jest.fn(),
}));

describe('MixOverview', () => {
  test('should render correctly', () => {
    const { asFragment } = render(
      <RecoilRoot>
        <MixOverview />
      </RecoilRoot>,
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
