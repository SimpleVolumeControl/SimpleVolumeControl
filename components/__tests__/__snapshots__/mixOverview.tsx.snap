// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MixOverview should render correctly 1`] = `
<DocumentFragment>
  <div
    class="container mx-auto grid grid-cols-[repeat(auto-fit,minmax(250px,1fr))] gap-4"
  />
</DocumentFragment>
`;
